de-lib/src/main/java/org/iplantc/de/client/services/FileEditorServiceFacade.java

```diff
@@ -17,7 +18,7 @@
      *  @param file desired manifest's file ID (path).
      * @param callback executes when RPC call is complete.
      */
-    void getManifest(File file, AsyncCallback<Manifest> callback);
+    void getManifest(File file, DECallback<Manifest> callback);
```

Comment:
From what I can tell, if I enable maintenance on tree-urls, I can still view a file (which calls the manifest endpoint) and the Data window stays enabled.  So we're good here 👍